packages/client/lib/client.ts

```diff
@@ -110,6 +112,14 @@ export class EthereumClient {
       this.config.logger.warn(`Server error: ${error.name} - ${error.message}`)
     })
     this.config.events.on(Event.SERVER_LISTENING, (details) => {
+      const networkDir = this.config.getNetworkDirectory()
+      // Write the transport into a file
+      try {
+        writeFileSync(`${networkDir}/${details.transport}`, details.url)
```

Comment:
Do we have an actual use case for this?  Feels like we're just arbitrarily writing something to disk and not using it anywhere.

Reply:
yes, i am using it to do a multi peer setup in shandong with script reading from file, as well as picking it up  to populate enodes